internal/webhook/external/artifactory.go

```diff
@@ -181,9 +182,12 @@ func (a *artifactoryWebhookReceiver) getHandler(requestBody []byte) http.Handler
 			)
 			return
 		}
-		logger = logger.WithValues("repoURL", repoURL)
+		logger = logger.WithValues(
+			"repoURL", repoURL,
+			"tag", payload.Data.Tag,
+		)
```

Comment:
Looks like this was already accounted for on line +136

```suggestion
```

Reply:
Rationale being:

1. follows existing convention
2. could be a useful means of debugging for a possible future url parsing/concatenation edge case we may not have accounted for

Reply:
You're right. Tag was and url wasn't. Your fix is more correct than my suggestion. 👍